filecoin/storage/adapter.go

```diff
@@ -31,10 +32,10 @@ type Adapter struct {
 
 // GetChainHead returns a tipset token for the current chain head
 func (a *Adapter) GetChainHead(ctx context.Context) (shared.TipSetToken, abi.ChainEpoch, error) {
-	fmt.Println("GetChainHead")
+	log.Info().Msg("GetChainHead")
```

Comment:
These I will probably remove when storage is more stable. This was mostly to help debug stuff.